tests/test_common_gpu.py

```diff
@@ -50,7 +50,7 @@
 )
 from peft.import_utils import is_bnb_4bit_available, is_bnb_available
 from peft.tuners.lora.config import LoraRuntimeConfig
-
+from peft.tuners.boft.layer import Linear, Conv2d
```

Comment:
Let's import `from peft.tuners import boft` and below use `boft.layer.Linear` and `boft.layer.Conv2d`.